### PR TITLE
vcsim: Use soapenv namespace as required by other clients

### PR DESCRIPTION
**Pull request by dougm**
With this set of changes, the pyvmomi/sample programs and rbvmomi+rvc
basics work against vcsim.

### CODE REVIEW

**Comment by dougm** [comment]
@lamw @jakerobinson tested these changes with pyvmomi and rbvmomi/rvc, haven't tried power cli yet.

File: pkg/vsphere/simulator/property_collector.go [hunk 1]
```diff
@@ -112,7 +112,11 @@ func fieldValue(rval reflect.Value, p string) (interface{}, error) {
 	fields := strings.Split(p, ".")
```
Comment (sgairo) [inline pkg/vsphere/simulator/property_collector.go]: ^^copyright needs a range :)

File: pkg/vsphere/simulator/property_collector_test.go [hunk 2]
```diff
@@ -339,5 +339,88 @@ func TestExtractEmbeddedField(t *testing.T) {
 	if ok {
```
Comment (sgairo) [inline pkg/vsphere/simulator/property_collector_test.go]: ^^ ditto

**Comment by jakerobinson** [comment]
@jrgarcia you should be trying this out :) Here's a recap and quickstart (thanks William) http://www.virtuallyghetto.com/2017/04/govcsim-neat-incubation-project-vcenter-server-esxi-api-based-simulator.html

**Comment by jrgarcia** [comment]
@jakerobinson You better believe I've already been looking at this. :smile:

**Pull request merged**